packages/cw-orch-fns-derive/src/execute_fns.rs

```diff
@@ -148,7 +148,6 @@ pub fn execute_fns_derive(input: DeriveInput) -> TokenStream {
 
     let derived_trait_impl = quote!(
         #[automatically_derived]
-        #[allow(undocumented)]
```

Comment:
Why did you remove this?

Reply:
`automatically_derived` covers it for impl blocks!

Reply:
Actually it won't warn on trait impls, since docs of it should be inside the trait itself